### PR TITLE
feat: KPI-25 Map domain

### PR DESCRIPTION
**Pull request by ridouku**
## Jira Link

[Go to Jira](https://kpinetwork.atlassian.net/browse/KPI-25)

## Types of changes

- [ ] Hotfix (Bug detected)
- [x] Feature (Issue from an Active Sprint)

## Checklist:

- [ ] Unit Test.
- [ ] Coverage.

### CODE REVIEW

File: infrastructure/variables.tf [hunk 1]
```diff
@@ -16,8 +18,14 @@ variable "region" {
...
 variable "aws_account_id" {
   description = "AWS account id"
+  sensitive = true
 }
 
+variable "root_domain_name" {
+  default = "kpinetwork.com"
+}
+
+
```
Comment (kellycastrof) [inline infrastructure/variables.tf:28]: Please, remove this line

File: infrastructure/certificates/acm.tf [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+resource "aws_acm_certificate" "cert" {
+  provider = aws.east
+  domain_name = "api.${var.domain}"
+  validation_method = "DNS"
+
+  lifecycle {
+    create_before_destroy = true
+  }
+//subject_alternative_names = var.cert_sans
```
Comment (kellycastrof) [inline infrastructure/certificates/acm.tf:20]: I think you can remove this comment.

File: infrastructure/modules.tf [hunk 3]
```diff
@@ -59,4 +61,22 @@ module "codebuild" {
...
+
+module "dns" {
+  source = "./dns/"
+  api_gateway_domain = module.network.api_gateway_domain
+  domain = var.root_domain_name
+  hosted_zone_id = aws_route53_zone.kpinetwork.id
+  domain_certificates = module.cert.domain_certificate
+}
+
+
```
Comment (kellycastrof) [inline infrastructure/modules.tf:74]: Remove extra empty lines

File: infrastructure/certificates/acm.tf (hunk 2 above)
Comment (ridouku) [reply infrastructure/certificates/acm.tf:20]: This line is useful when someone else wants to renew the certificates, I think is better leave it

**Review by kellycastrof — APPROVED** [review]
👍

**Pull request merged**